cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,22 @@
 /// <reference types="cypress" />
 
-describe('Sign In page', () => {
-  it('should provide an ability to log in', () => {
+describe('Sign In', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
 
+  it('should Sign In', () => {
+    const username = 'test123456dasdas';
+    const email = 'lotar@mailinator.com';
+    const password = 'testqa123!';
+    cy.contains('Sign in').click()
+    cy.get('h1').should('contain', 'Sign In')
+    cy.url().should('include', '/login')
+    cy.get('[placeholder="Email"]').type(email)
+    cy.get('[placeholder="Password"]').type(password)
+    cy.get('.btn').click()
+    cy.get('[class="nav-link"]').should('contain', username)
```

Comment:
The assertion is checking for a hardcoded username. It should check for the username that was used to sign in.